script/main.js

```diff
@@ -249,10 +249,23 @@ class App extends Component {
     this.setState({versions: e.target.value.split(',')});
   }
 
+  normalizeVersions(versions) {
+    return versions.map(version => {
+      if (parseInt(version) < 5) {
+        throw new Error(`Unsupported Lighthouse version (${version})`);
+      }
+      // In the future we might want a more generalized `version % 2 === 1` thing, but for now, hardcode the change.
+      if (parseInt(version) === 7) return 6..toString();
```

Comment:
hello ruby my old friend

Reply:
Whoa range operator

Reply:
😀 not range operator. it's just how you call methods on a number literal.

one could do `123.45.toLocaleString()`  but if you're calling on an integer, you'll need two periods: `123..toLocaleString()`. (If you try `123.toLocaleString()`, you'll get a SyntaxError. 

i dont remember where i picked this up, but it's some esoteric JS knowledge that's been tucked away in my brain. :p